packages/api/lambdas/kinesis-consumer.js

```diff
@@ -3,12 +3,16 @@
 'use strict';
 
 const Ajv = require('ajv');
-
-const log = require('@cumulus/common/log');
+const {
+  aws: { sns },
+  log
+} = require('@cumulus/common');
 const Rule = require('../models/rules');
 const messageSchema = require('./kinesis-consumer-event-schema.json');
 const sfSchedule = require('./sf-scheduler');
 
+const fallbackArn = process.env.FallbackTopicArn;
```

Comment:
Since this is just used in `publishRecordToFallbackTopic`, it might be better to just have it in there instead of as a global const.